apps/server/src/modules/school/controller/dto/migration.response.ts

```diff
@@ -1,10 +1,21 @@
+import { IsDate } from 'class-validator';
+
 export class MigrationResponse {
-	oauthMigrationPossible!: boolean;
+	@IsDate()
+	oauthMigrationPossible: Date | undefined;
+
+	@IsDate()
+	oauthMigrationMandatory: Date | undefined;
+
+	@IsDate()
+	oauthMigrationFinished: Date | undefined;
 
-	oauthMigrationMandatory!: boolean;
+	enableMigrationStart!: boolean;
```

Comment:
What is this for? Could you explain in the ApiProperty description? I saw it will set when there is a official school number